### ISSUE TITLE
redis output sends empty `@metadata.beat` on 6.3.0

### ISSUE THREAD

**Issue by jsvd** [elastic/beats#7389]
Replicating is easy with a redis server instance up, enabling redis output in packetbeat 6.3.0, then checking the data on redis:

```
127.0.0.1:6379> LPOP packetbeat
"{\"@timestamp\":\"2018-06-21T11:54:02.310Z\",\"@metadata\":{\"beat\":\"\",\"type\":\"doc\",\"version\":\"6.3.0\"},\"server\":\"\",\"client_ip\":\"192.168.1.76\",\"client_port\":53222,\"host\":{\"name\":\"Joaos-MBP-5.lan\"},
```

we can see that  `@metadata.beat` value is `""`

**Comment by marceliq** [comment]
This is also a problem in 6.2.4.

**Comment by urso** [comment]
The fix and other improvements to the redis output should become available with 6.4

**Comment by anjia0532** [comment]
@urso thanks.
